intel_extension_for_transformers/llm/runtime/graph/models/llama/llama.cpp

```diff
@@ -120,6 +135,13 @@ static bool llama_model_eval_internal(model_context& lctx, const model_token* to
   ne_set_name(embd, "embd");
   memcpy(embd->data, tokens, N * ne_element_size(embd));
 
+#ifdef NE_TP_MODEL
+  if (enable_tp) {
+    // need to broadcast the ids
+    broadcast(p_ctx, (float*)embd->data, N * ne_element_size(embd));
+  }
+#endif
```

Comment:
Is it necessary to broadcast the embd data? I think all ranks should have the same embed upon here.